Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Prepare the boot build image and dependencies
+FROM clojure:boot-alpine AS boot
+RUN apk add --update git npm
+RUN npm install -g yarn shadow-cljs
+
+# Copy source files into the docker container
+WORKDIR /app
+COPY . ./
+
+# Update game data to the latest release
+RUN git submodule update --init
```

Comment:
In order for this to work, we need to switch to `https` URL of the submodule. Otherwise this command results in a `cannot run ssh: No file or directory` error (since current submodule URL is ssh-based), and if I install `openssh-client-default` package with `apk`, I receive `Host key verification failed` (which I didn't really investigate, maybe it can be solved).

Anyway, it would be simpler just to call `git submodule set-url resources/game https://github.com/endless-sky/endless-sky.git`